components/net/indexeddb/engines/sqlite.rs

```diff
@@ -0,0 +1,513 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+use std::path::{Path, PathBuf};
+
+use net_traits::indexeddb_thread::{
+    AsyncOperation, AsyncReadOnlyOperation, AsyncReadWriteOperation, CreateObjectResult,
+    IndexedDBKeyRange, KeyPath, PutItemResult,
+};
+use sea_orm::prelude::*;
+use sea_orm::sea_query::IntoCondition;
+use sea_orm::{Condition, Database, IntoActiveModel, NotSet, Set};
+use tokio::sync::oneshot;
+
+use crate::async_runtime::HANDLE;
+use crate::indexeddb::engines::{KvsEngine, KvsTransaction, SanitizedName};
+use crate::indexeddb::idb_thread::IndexedDBDescription;
+
+mod database_model;
+mod object_data_model;
+mod object_store_index_model;
+mod object_store_model;
+// mod serialize;
+mod serialize {
```

Comment:
What's the difference between the serialization in this file vs. the one in the script crate?